### DISCUSSION TITLE
New function: read two books simultaneously!

### DISCUSSION THREAD

**Discussion by simuleite** [koreader/koreader#11432] [Ideas]
I am a Chinese reader. Sometimes I must read some English docs but it's kind of hard.  What if KOReader could read two books simultaneously, one in Chinese and the other in English? It will be really nice! I know there's a function called "Two Columns".   It would be wonderful if users could read two books in two columns.

**Comment by Frenzie** [comment]
It does sound interesting, but I wouldn't count on something like that anytime soon. ;-)

**Comment by Vuizur** [comment]
The program worddumb does something very similar and adds translations to the words: https://github.com/xxyzz/WordDumb 

I personally like it even more than parallel texts.

**Comment by Name2889** [comment]
Sorry, cannot you just use dictionary in Koreader? I am also not native speaker, and when I do not understand some words, I long tap on the word and it returns translation. (just in case you were not aware of this functionality)

**Reply by Frenzie** [reply]
It can be quite nice to read something in a language you partially know and one you know well side by side. It's quite common for Latin for example, but also other classic works. :)
